plugins/coral-plugin-offtopic/client/containers/OffTopicCheckbox.js

```diff
@@ -0,0 +1,11 @@
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {addTag, removeTag} from 'coral-plugin-commentbox/actions';
```

Comment:
We need to provide this actions through the `coral-plugin-api` in a folder called `alpha` or `beta` if we are more confident that it doesn't change.